float32 max_trans_velocity
---
